tests/unittests/utils/test_mapanything.py

```
@@ -0,0 +1,1018 @@
+import torch
+from PIL import Image
+from unittest.mock import patch, MagicMock
+
+import fiftyone.core.labels as fol
+
+
+# ---------------------------------------------------------------------------
+# Helpers
+# ---------------------------------------------------------------------------
+
+def _make_model(output_type="depth"):
```

Ruff (0.14.14)

[warning] Missing return type annotation for private function `_make_model`

(ANN202)